packages/core/src/types/lifecycle.ts

```diff
@@ -395,6 +395,35 @@ export type HyperlaneTransferInfo = {
   txs: HyperlaneTransferTransactions;
 };
 
+export type OPInitTransferState =
+  | "HYPERLANE_TRANSFER_UNKNOWN"
```

Comment:
This has hyperlane in it, imagine not intentional

Reply:
good catch